app/Helpers/helpers.php

```diff
@@ -26,20 +30,20 @@ function setTimeShort($value)
         $split = explode(' ', $time);
         switch ($split[1]) {
             case __('settings.time.full.seconds'):
-                return $split[0] . ' ' . __('settings.time.short.seconds');
-                break;
+            return $split[0] . ' ' . __('settings.time.short.seconds');
+            break;
             case __('settings.time.full.minutes'):
-                return $split[0] . ' ' . __('settings.time.short.minutes');
-                break;
+            return $split[0] . ' ' . __('settings.time.short.minutes');
+            break;
             case __('settings.time.full.hours'):
-                return $split[0] . ' ' . __('settings.time.short.hours');
-                break;
+            return $split[0] . ' ' . __('settings.time.short.hours');
+            break;
             case __('settings.time.full.days'):
-                return $split[0] . ' ' . __('settings.time.short.days');
-                break;
+            return $split[0] . ' ' . __('settings.time.short.days');
+            break;
             default:
-                return $split[0] . ' ' . __('settings.time.short.weeks');
-                break;
+            return $split[0] . ' ' . __('settings.time.short.weeks');
```

Comment:
**Reporter**: CHECKSTYLE</br>**Rule**: PSR2.ControlStructures.SwitchDeclaration.BreakIndent</br>**Severity**: error</br>**File**: app/Helpers/helpers.php</br>**Line**: 45</br>Terminating statement must be indented to the same level as the CASE body